python/o80_pam/run_robot.py

```diff
@@ -16,16 +16,15 @@
 def _set_min_pressures(config, pam_config):
 
     frontend = o80_pam.FrontEnd(config.segment_id)
-    for dof in range(config.nb_dofs):
+    for dof in range(4):
```

Comment:
I am a bit confused about this.  Do I understand the code below in `configure()` correctly, that nb_dof is still configurable in case the robot is not pamy2.  If this is the case, the value should probably not be hard-coded here?

Reply:
yes, code was well templated in this regards for pamy1 but not for pamy2.
Still, this hard-coded "4" is not good.
I updated so that the instance of config has the attribute nb_dofs for both robots